EventListener/AcbFileListener.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Sherlockode\AdvancedContentBundle\EventListener;
+
+use FOS\UserBundle\FOSUserEvents;
+use Sherlockode\AdvancedContentBundle\Event\AcbFileEvent;
+use Sherlockode\AdvancedContentBundle\Event\AcbFilePostValidate;
+use Sherlockode\AdvancedContentBundle\Event\AcbFilePreSubmitEvent;
+use Sherlockode\AdvancedContentBundle\Manager\UploadManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class AcbFileListener implements EventSubscriberInterface
+{
+    /**
+     * @var UploadedFile
+     */
+    private $file;
+
+    /**
+     * @var string
+     */
+    private $fileName;
+
+    /**
+     * @var UploadManager
+     */
+    private $uploadManager;
+
+    /**
+     * @param UploadManager $uploadManager
+     */
+    public function __construct(UploadManager $uploadManager)
+    {
+        $this->uploadManager = $uploadManager;
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            AcbFilePreSubmitEvent::NAME => 'onPreSubmit',
+            AcbFilePostValidate::NAME => 'onPostValidate',
+        ];
+    }
+
+    /**
+     * @param AcbFilePreSubmitEvent $event
+     *
+     * @return void
+     */
+    public function onPreSubmit(AcbFilePreSubmitEvent $event): void
+    {
+        $this->file = $event->getUploadedFile();
```

Comment:
If we use a property to store the file, we cannot process multiple file uploads in the same HTTP request. Would an array of uploaded files be a better option?